search-enterprise/latest/using/unsupported.md

```diff
@@ -68,11 +66,11 @@ Search Enterprise focuses on the most commonly used features, and in some cases
 The following are examples of how to use different fields for queries:
 
 | Old Query | Replacement Query |
-| -------- | ----------------- |
+| --------- | ----------------- |
 | QNAME:'comment' | TYPE:'fm:post' |
 | PNAME:'0/wiki' | PATH:'//cm:wiki/*' |
-| NPATH:'2/Company Home/Sites/swsdp' | PATH: '/app:company_home/st:sites/cm:swsdp//*' |
-| PATH:'//cm:CategoryA/member' | PATH: '//cm:CategoryA/*' AND !TYPE:'cm:category' |
+| NPATH:'2/Company Home/Sites/swsdp' | PATH: '/app:company_home/st:sites/cm:swsdp/*' |
+| ANAME:'0/cdefb3a9-8f55-4771-a9e3-06fa370250f6' | PARENT:'cdefb3a9-8f55-4771-a9e3-06fa370250f6' |
 
 > **Note:** Secondary paths and secondary parents are not supported at this time, so there may still be some differences if these are in use.
```

Comment:
That note is not true anymore. With `ESC 4.0.0` we are introducing secondary paths and parents.

Reply:
Removed note